test/protocol.test.js

```diff
@@ -0,0 +1,9 @@
+'use strict';
+/* globals describe, it */
+const TestProtocol = require('../test_protocol/');
+
+describe('protocol test runners', async function() {
+    it('should run all protocol test scripts', async function() {
+        await TestProtocol()
```

Comment:
Do we not want to hook it into the tests a little bit better, so each test becomes `it` and gets reported correctly by the test software?